Dockerfile.ci

```diff
@@ -92,6 +94,19 @@ RUN mkdir -p /opt/toolchains && \
 	zephyr-sdk-${ZSDK_VERSION}/setup.sh -t all -h -c && \
 	rm zephyr-sdk-${ZSDK_VERSION}_linux-${HOSTTYPE}.tar.xz
 
+# Install FVP
+RUN mkdir -p /opt/fvps && \
+    cd /opt/fvps && \
+    wget ${WGET_ARGS} https://developer.arm.com/-/media/Files/downloads/ecosystem-models/FVP_Base_RevC-2xAEMvA_${FVP_BASE_REVC_VERSION}_Linux64.tgz && \
+    tar xf FVP_Base_RevC-2xAEMvA_${FVP_BASE_REVC_VERSION}_Linux64.tgz && \
+    ln -s /opt/fvps/Base_RevC_AEMvA_pkg/models/Linux64_GCC-9.3/FVP_Base_RevC-2xAEMvA /usr/local/bin && \
+    rm FVP_Base_RevC-2xAEMvA_${FVP_BASE_REVC_VERSION}_Linux64.tgz && \
+    wget ${WGET_ARGS} https://developer.arm.com/-/media/Files/downloads/ecosystem-models/FVP_Base_AEMv8R_${FVP_BASE_AEMV8R_VERSION}_Linux64.tgz && \
+    tar xf FVP_Base_AEMv8R_${FVP_BASE_AEMV8R_VERSION}_Linux64.tgz && \
+    ln -s /opt/fvps/AEMv8R_base_pkg/models/Linux64_GCC-9.3/FVP_BaseR_AEMv8R /usr/local/bin && \
+    rm FVP_Base_AEMv8R_${FVP_BASE_AEMV8R_VERSION}_Linux64.tgz && \
+    apt-get install -y telnet xterm
```

Comment:
Unless the FVP binaries are architecture-agnostic, they should be encapsulated in `if [ "${HOSTTYPE}" = "x86_64" ]; then` (note that this runs for x86 and AArch64 image builds).

Also are there AArch64 host binaries for FVP? If so, the script should be updated to download AArch64 binaries for AArch64 host.

Reply:
There is only for `Base RevC-2xAEMvA`. `Base_AEMv8R` is only available for `Linux64`. To keep it as a all-or-nothing, I’ll gate it all with `if [ "${HOSTTYPE}" = "x86_64" ]; then` if that’s ok for you.

Reply:
That is ok for now. Is there any plan to release AArch64 host version in the future? Because the new Zephyr CI runners (WIP) are going to be both x86 and AArch64 (jobs can be scheduled onto both x86 and AArch64 nodes) and feature parity across those two architectures would be desired.

Reply:
Corstone-300, Corstone-310 and Armv8-A AEM FVPs have beta version for aarch64 hosts, I don't know the plans for the others.